test/CollectionIntegrationTest.php

```diff
@@ -48,9 +48,6 @@ public function setUp()
 
     public function setUpHelpers()
     {
-        if (isset($this->helpers)) {
```

Comment:
Removing these returns so this setup is done for each test run, eliminating any sort of global state issues.  (there are only two tests in this file)